### ISSUE TITLE
Salivary Basal cell adenoma

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10195]
**Preferred term label:**

**Synonyms**
Also called monomorphic adenoma, basaloid salivary gland adenoma 

**Definition (free text, please give PubMed ID)**
 Zhao X, Wei S. Basal cell adenoma. PathologyOutlines.com website. https://www.pathologyoutlines.com/topic/salivaryglandsbasalcelladenoma.html. 
Benign salivary tumor composed of nests and cords of small basaloid cells with inner ductal epithelial cells 

PMID: 26500869 "Proteus syndrome is a rare condition with a wide spectrum of abnormalities. It is characterized by hamartomatous malformations involving multiple organs. Serious complications may ensue, such as pulmonary embolism, cystic lung disease, and various neoplasms such as **parotid adenomas**, ovarian cystadenomas, and meningiomas. 

PMID: 22876373 (...) "Tumors, most commonly meningiomas. Ovarian cystadenomas, breast cancer, parotid monomorphic 
adenoma, mesothelioma, and others have also been reported."

**Parent term (use hpo.jax.org/app)**
Salivary gland neoplasm HP:0100684

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:79493  Brooke-Spiegler syndrome
 ORPHA:744      Proteus syndrome

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000007
- label: Salivary basal cell adenoma
- parents: HP:0100684
- pmid: PMID:26693465
- def: A rare neoplasm consisting of a monomorphic population of basaloid epithelial cells, and it accounts for approximately 1-2% of all salivary gland tumors.
- comment: Thee most frequent location of basal cell adenoma (BCA) of the salivary gland is the parotid gland. It usually appears as a firm and mobile slow-growing mass. Histologically, isomorphic cells in nests and interlaced trabecules with a prominent basal membrane are observed. In contrast to pleomorphic adenoma, it tends to be multiple and its recurrence rate after surgical excision is high.

- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10195
- synonyms:
